### DISCUSSION TITLE
Form column default value [Aug 18]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3351] [Discord Threads]
> 🙋‍♂️ **AlexU**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Not supported at the moment  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1009285756889604096)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
